### DISCUSSION TITLE
I can not mount the fuse filesystem, does anybody know why?

### DISCUSSION THREAD

**Discussion by liusitan** [v6d-io/v6d#834] [Q&A] [answered]
<img width="637" alt="image" src="https://user-images.githubusercontent.com/20542539/178128360-8d79fc6a-c71a-459d-9171-487e41bc0cb2.png">
Hi, I found that I can not mount the fuse filesystem implementation in the repo, it reports to me that _Numerical result out of range_. The error is surprising to me because at the beginning of every fuse operation, there comes a LOG operation, at least it should output some log operation. Anybody can lend me a hand?

**Comment by sighingnow** [comment, marked as answer]
It should be caused by an invalid return code.

**Comment by liusitan** [comment]
solved, yes, I deregistered fuse_access and now cd works.
